playbook.md

```diff
@@ -1630,6 +1630,110 @@ If you think this might be for you, please apply through our jobs page.
 
 ## Working arrangements
 
+### dxw Support Structures
+
+#### Practical day to day support
+
+Anytime you need to order something, notice a problem in the office (e.g. a broken chair or dripping tap) or you need some help with your laptop or the printer - there is a Slack channel for it. If in doubt, please speak to the Commercial Operations team and they’ll point you in the right direction.
+
+```
+#dxw-facilities
+#dxw-purchasing
+#dxw-travel
+#dxw-techsupport
+```
+
+#### Helpers
+
+When you join dxw you will be assigned a helper. This person will work closely with you for at least your first 4 months. They will do a similar job to you, help you meet members of the wider team, catch up with you on at least a 3/6/12 week basis to ensure you are receiving everything you need, and will NEVER get bored of your questions!
+
+If you feel that you’re not getting as much time with your helper as you’d like, please speak to the People Manager.
+
+As a helper, you will be vital in ensuring you have the regular catch ups - if you are finding it hard to schedule them, please do talk to Bus Ops.
+
+For more information on being a helper, please see the [checklist](https://docs.google.com/document/d/1Dc3IRZ_ze7QzMQTHO70ZC6hU8cig5ds7LN2MoHmPBe8/edit) or speak to the People Manager.
+
+#### Buddies
+
+Once you have passed your 4 month probation, we would like you to choose a buddy within the company to support you.
+
+We recommend this person **doesn’t** work directly with you, someone in a different role and that you don’t work closely with on projects, as this will give you a second pair of ears and a whole new set of experiences to draw on.
+
+This is not mandatory, just recommended and similarly not everyone may wish to be a buddy - don’t be afraid to ask or say no. It can be reciprocal but doesn’t have to be.  Also, if you feel that a change of buddy would benefit you for whatever reason or that you’re no longer able to be a buddy, this should also not be an awkward situation. If you have any concerns, have a chat with the People Manager.
+
+If you want to know more about how the buddy system works, how often to meet and suggested discussion topics, see the buddy [crib sheet](https://docs.google.com/document/d/1PBKwPLFsGWUUzeYAObcFLQJyTPknkLhm2HrbuHwxelU/edit?usp=sharing). This shouldn’t be a huge time commitment, ideally just a pleasant Friday meet up.
+
+#### HR
+
+The bi-weekly HR surgeries are there to give an opportunity for members of the team to discuss matters they might not feel able to raise with buddies/delivery leads/line managers or that are specialist to the HR function in dxw.
+
+The list is not exhaustive but some topics might include:
+
+* Holiday
+* Sickness
+* Pay
+* Expenses
+* Parental Leave
+* Accessibility
+* Probation
+* Pensions
+* Occupational health issues
+* Contractual matters
+* Working patterns/locations
+* Training and career progression
+* Diversity
```

Comment:
What does "diversity" as a topic in an HR surgery mean? Is this really "inclusivity"?

Reply:
If anyone had any concerns with regard to diversity from an HR point of view - including inclusivity as well would be a great idea though.